packages/experimental/Button/src/Button.tsx

```diff
@@ -47,12 +47,13 @@ export const Button = compose<ButtonType>({
 
     // now return the handler for finishing render
     return (final: ButtonProps, ...children: React.ReactNode[]) => {
-      const { icon, iconPosition, content, ...mergedProps } = mergeProps(button.props, final);
+      const { icon, iconPosition, ...mergedProps } = mergeProps(button.props, final);
       return (
         <Slots.root {...mergedProps}>
           {icon && iconPosition === 'before' && <Slots.icon {...iconProps} />}
-          {content && <Slots.content key="content">{content}</Slots.content>}
-          {children}
+          {Object.keys(children).map((key) =>
```

Comment:
I would consider using the React children iterators just to make sure we're handling edge cases appropriately:

https://reactjs.org/docs/react-api.html#reactchildren

Reply:
done